documentation/source/users/arkane/input.rst

```diff
@@ -421,6 +421,44 @@ for specifying the same ``rotors`` entry are commented out)::
 
 Note that the atom labels identified within the rotor section should correspond to the indicated geometry.
 
+Multidimensional Hindered Rotors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+For multidimensional hindered rotors, ``geometry``, ``frequencies`` need to be specified in the same way described in detail in the last section. However,
+rotor files may specified by either a 1D rotor scan file as in the last section (1D rotors only) or as a directory. 
+The directory must contain a list of energy files in the format ``something_name_rotorAngle1_rotorAngle2_rotorAngle3(...).log``.  
```

Comment:
Doesn't have to be implemented right away, just wanted to hear your thought:
What if we have a single Gaussian output file generated by passing two scan directives (two `D       2       1       7      14 S  36 10.000`)? Can we read the energies and dihedral angles from it instead of having multiple files?

Reply:
We don't have a parser for it now, but once we write one it'll be easy to add.

Reply:
There are a couple other things to bring up though. Doing it in one output file means you can't run the points in parallel.  It also will give you different results, since each point becomes a function of the last point in the scan.

Reply:
Since we parallelize Gaussian itself, the first point might not be a big issue. For the second point, I agree, but that might (?) be a good thing (closer to reality)

Reply:
The native Gaussian parallelization is much less efficient than parallelizing each optimization all at once.